src/ts/ConvivaAnalytics.ts

```diff
@@ -196,7 +202,7 @@ export class ConvivaAnalytics {
    * @param eventAttributes a string-to-string dictionary object with arbitrary attribute keys and values
    */
   public sendCustomApplicationEvent(eventName: string, eventAttributes: EventAttributes = {}): void {
-    this.client.sendCustomEvent(Conviva.Client.NO_SESSION_KEY, eventName, eventAttributes);
+    // this.client.sendCustomEvent(Conviva.Client.NO_SESSION_KEY, eventName, eventAttributes);
```

Comment:
I'll remove the code block. I am not sure the proper way to send custom events. I posted on the conviva forum asking for a public API because their tutorial is lacking. The public documentation they have doesn't mention anything about custom events

Reply:
Makes sense, thanks for the feedback.